bundle/manifests/opendatahub-operator.clusterserviceversion.yaml

```diff
@@ -1712,7 +1713,7 @@ spec:
                       fieldPath: metadata.namespace
                 - name: DEFAULT_MANIFESTS_PATH
                   value: /opt/manifests
-                image: REPLACE_IMAGE:latest
+                image: quay.io/wenzhou/opendatahub-operator:2.15.0
```

Comment:
is this expected ?

Reply:
will revert it once done testing after remove this CM

Reply:
done